### ISSUE TITLE
Required attribute

### ISSUE THREAD

**Issue by scrappergus** [onesine/react-tailwindcss-datepicker#153]
I've added an option to include a 'required' attribute on the input. It's a simple update, will submit pull request next if you'd like.

**Comment by alphabetek** [comment]
@scrappergus any news? It seems this option is still missing, will you submit a pull request?

**Comment by scrappergus** [comment]
Yes, I will, the guidelines have previously asked that we ask first.

**Comment by netgfx** [comment]
Will this PR get merged?

**Comment by scrappergus** [comment]
No idea, @onesine? You can use my fork until then: https://github.com/scrappergus/react-tailwindcss-datepicker
